consensus/consensus_v2.go

```diff
@@ -111,9 +111,13 @@ func (consensus *Consensus) announce(block *types.Block) {
 	consensus.block = encodedBlock
 	consensus.blockHeader = encodedBlockHeader
 	msgToSend := consensus.constructAnnounceMessage()
+	// TODO Finish using this refactored way
+	// msgToSend := consensus.construct(quorum.Announce)
 
 	// save announce message to FBFTLog
+	// msgPayload, _ := proto.GetConsensusMessagePayload(msgToSend.Bytes)
```

Comment:
why remove it?

Reply:
I didn't, its a copy commented as as reference while I was refactoring the API, call it preserved, next line.